Dockerfile

```diff
@@ -22,41 +37,94 @@ RUN set -ex && \
-
-# Download and install glibc
-# NOTE: Keep an eye on the issue https://github.com/aws/aws-cli/pull/6352 regarding installation of the AWS CLI from source in order to avoid
-# the need of installing glibc
-# With newer versions of Alpine (e.g. 3.16, used in Terraform 1.3.5 container image) this error can show up:
-# ERROR: glibc-2.34-r0: trying to overwrite etc/nsswitch.conf owned by alpine-baselayout-data-3.2.0-r23
-# if this happen, use the --force-overwrite flag in apk add
-# issue here https://github.com/sgerrand/alpine-pkg-glibc/issues/185
-RUN curl -sL "https://alpine-pkgs.sgerrand.com/sgerrand.rsa.pub" -o /etc/apk/keys/sgerrand.rsa.pub \
-        && curl -sLO "https://github.com/sgerrand/alpine-pkg-glibc/releases/download/${GLIBC_VERSION}/glibc-${GLIBC_VERSION}.apk" \
-        && curl -sLO "https://github.com/sgerrand/alpine-pkg-glibc/releases/download/${GLIBC_VERSION}/glibc-bin-${GLIBC_VERSION}.apk" \
-        && apk add --no-cache --force-overwrite \
-        glibc-${GLIBC_VERSION}.apk \
-        glibc-bin-${GLIBC_VERSION}.apk \
-        && rm glibc-${GLIBC_VERSION}.apk glibc-bin-${GLIBC_VERSION}.apk
+        oathtool \
+        wget \
+        python3 \
+        git
+        # oath-toolkit-oathtool \
+
+#RUN ln -s /usr/bin/python3 python
+
+################################################################
+################################################################
+
+FROM base AS leverage-base
+
+################################
+# Versions
+################################
+
+ARG TERRAFORM_VERSION
+ARG GLIBC_VERSION
+ARG AWSCLI_VERSION
+ARG AWSVAULT_VERSION
+ARG HCLEDIT_VERSION
+
+################################
+# Install Terraform
+################################
+
+# Download terraform for linux
+RUN wget https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/terraform_${TERRAFORM_VERSION}_linux_amd64.zip
+
+# Unzip
+RUN unzip terraform_${TERRAFORM_VERSION}_linux_amd64.zip
+
+# Move to local bin
+RUN mv terraform /usr/local/bin/
+RUN ln -s /usr/local/bin/terraform /bin/terraform
+# Check that it's installed
+RUN terraform --version
+
+################################
+# Install AWS CLI
+################################
 
 # Install AWS CLI v2
 RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64-${AWSCLI_VERSION}.zip" -o awscliv2.zip \
         && unzip awscliv2.zip \
         && ./aws/install \
         && rm -rf awscliv2.zip aws
 
+################################################################
+################################################################
+
+FROM leverage-base AS leverage-toolbox
+
+################################
+# Versions
+################################
+
+ARG TERRAFORM_VERSION
+ARG GLIBC_VERSION
+ARG AWSCLI_VERSION
+ARG AWSVAULT_VERSION
+ARG HCLEDIT_VERSION
+
+################################
+# Install 
+################################
+
 # Add aws-vault
 RUN curl -LO "https://github.com/99designs/aws-vault/releases/download/${AWSVAULT_VERSION}/aws-vault-linux-amd64" \
```

Comment:
@juanmatias @angelofenoglio @exequielrafaela Can't remember when we started using this? Do you recall?

Reply:
Yeah, I don't think it's used anywhere. Let's remove it and see what breaks (since rolling back should be pretty easy).

Reply:
done